### DISCUSSION TITLE
is there a way to import vector graphics ?

### DISCUSSION THREAD

**Discussion by Zelmoghazy** [xournalpp/xournalpp#3836] [Q&A] [answered]
when I import an svg file it is rasterized and pixelated , is there a way to import vector graphics without this happening  ?

**Comment by Technius** [comment, marked as answer]
Unfortunately, vector graphics are not supported properly at the moment. We are planning to add proper support for vector images when the new file format is finalized (which is a long way out--see #937 for progress).
